.idea/codeStyles/Project.xml

```diff
@@ -1,13 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="OTHER_INDENT_OPTIONS">
```

Comment:
(Heads up: remove the IDE file changes)

Reply:
oops - will remove. Actually, I think I'll split into multiple PRs once it's ready.